ansible/roles/airflow/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+- name: create required directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    owner: "{{ cchq_user }}"
+    group: "{{ cchq_user }}"
+    mode: 0755
+    state: directory
+  with_items:
+    - "{{ airflow_home }}"
+    - "{{ airflow_code_dir }}"
+
+- name: Pull full Airflow source
+  git:
+    repo: "{{ airflow_repository }}"
+    dest: "{{ airflow_code_dir }}"
+    version: "{{ airflow_version }}"
+    recursive: yes
+    accept_hostkey: yes
+    update: yes
+  become: true
+
+
+- name: install pip requirements
+  become: true
+  pip:
+    requirements: "{{ airflow_code_dir }}/requirements.txt"
+    virtualenv: "{{ airflow_virtualenv }}"
+    chdir: "{{ airflow_code_dir }}"
+
+- name: chown clone
```

Comment:
probably not necessary if you specify `become_user: "{{ cchq_user }}"` on some of the tasks above